spacq/devices/keithley/dm2100.py

```diff
@@ -0,0 +1,119 @@
+import logging
+log = logging.getLogger(__name__)
+
+from spacq.interface.resources import Resource
+from spacq.tool.box import Synchronized
+
+from ..abstract_device import AbstractDevice
+from ..tools import quantity_wrapped
+
+"""
+Keithley 2100 Digital Multimeter
+
+Obtain measurements from the multimeter.
+"""
+
```

Comment:
I would move this to the top of the file. Sphinx can then use that as a module docstring